crates/rpc/rpc/src/eth/sim_bundle.rs

```diff
@@ -434,11 +434,11 @@ where
             .unwrap_or(DEFAULT_SIM_TIMEOUT);
 
         let bundle_res =
-            tokio::time::timeout(timeout, Self::sim_bundle(self, request, overrides, true))
+            tokio::time::timeout(timeout, Self::sim_bundle_inner(self, request, overrides, true))
                 .await
                 .map_err(|_| {
-                EthApiError::InvalidParams(EthSimBundleError::BundleTimeout.to_string())
-            })?;
+                    EthApiError::InvalidParams(EthSimBundleError::BundleTimeout.to_string())
+                })?;
```

Comment:
imo we can even get rid of the entire timeout handling here because this shouldn't take >1s right?